tsup.config.ts

```diff
@@ -0,0 +1,19 @@
+import { defineConfig } from 'tsup';
+
+export default defineConfig({
+  entry: [
+    'src/index.ts',
+    'src/bin/index.ts',
+    'src/types/index.ts',
+    'src/schema/index.ts',
+    'src/utils/index.ts',
+  ],
+  format: 'esm',
+  dts: true,
+  esbuildOptions(options) {
+    // This is required in order to make `typescript` work with `esbuild`.
+    // Otherwise, you will get:
+    // Error: "Dynamic required of 'fs' is not supported".
```

Comment:
```suggestion
    // Error: "Dynamic require of 'fs' is not supported".
```

I assume this is a typo, or...?

Reply:
Great catch!